draft-ZIP-0226.rst

```diff
@@ -27,27 +27,33 @@ The term "Orchard" in this document is to be interpreted as described in ZIP 224
 
 We define the following additional terms:
 
-- Asset: A type of note that can be transferred on the Zcash block chain. ZEC is the default (and currently the only defined) asset for the Zcash mainnet.
-- Zcash Shielded Asset: an asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard anonymity pool.
-- Wrapped Asset: a ZSA asset with native issuance defined outside the Zcash block chain.
+- Asset: A type of note that can be transferred on the Zcash block chain.
+
+  - ZEC is the default (and currently the only defined) asset for the Zcash mainnet.
+
+- Zcash Shielded Asset: an Asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard shielded pool.
+- Wrapped Asset: a Zcash Shielded Asset with native issuance defined outside the Zcash block chain.
+- Native Asset: a Zcash Shielded Asset created uniquely within the Zcash block chain.
+- Split Note: a Zcash Shielded Asset spend note that is used within an Action to ensure the output note of that Action is of a specific type. The Split Note is not spent in the transaction.
+- Split Action: a Zcash Shielded Asset Action that contains a Split Note.
```

Comment:
```suggestion
- Split Input: an Action input identifying a Zcash Shielded Asset note, in order to ensure that the output note of that Action is of a validly issued type, when there is no corresponding real input note to spend. The Action does not spend the note identified by the Split Input.
- Split Action: a Zcash Shielded Asset Action that contains a Split Input.
```

Reply:
Suggestion incorporated in [PR#649](https://github.com/zcash/zips/pull/649).